src/main.cpp

```diff
@@ -598,48 +595,57 @@ void GetGlobalStatus()
 
         // It is necessary to assign a local variable for ETTS to avoid an occasional deadlock between the lock below,
         // the lock on cs_main in GetEstimateTimetoStake(), and the corresponding lock in the stakeminer.
-        double dETTS = GetEstimatedTimetoStake()/86400.0;
+        double dETTS = GetEstimatedTimetoStake() / 86400.0;
+
         LOCK(GlobalStatusStruct.lock);
-        { LOCK(MinerStatus.lock);
-        GlobalStatusStruct.blocks = ToString(nBestHeight);
-        GlobalStatusStruct.difficulty = RoundToString(PORDiff,3);
-        GlobalStatusStruct.netWeight = RoundToString(GetEstimatedNetworkWeight() / 80.0,2);
-        //todo: use the real weight from miner status (requires scaling)
-        GlobalStatusStruct.coinWeight = sWeight;
-        GlobalStatusStruct.magnitude = RoundToString(boincmagnitude,2);
-        GlobalStatusStruct.ETTS = RoundToString(dETTS,3);
-        GlobalStatusStruct.ERRperday = RoundToString(boincmagnitude * NN::Tally::GetMagnitudeUnit(GetAdjustedTime()),2);
-        GlobalStatusStruct.cpid = NN::GetPrimaryCpid();
-        GlobalStatusStruct.poll = std::move(current_poll);
-
-        GlobalStatusStruct.status = msMiningErrors;
-
-        if(MinerStatus.WeightSum)
-            GlobalStatusStruct.coinWeight = RoundToString(MinerStatus.WeightSum / 80.0,2);
-
-        GlobalStatusStruct.errors.clear();
-        std::string Alerts = GetWarnings("statusbar");
-        if(!Alerts.empty())
-            GlobalStatusStruct.errors += _("Alert: ") + Alerts + "; ";
-
-        if (PORDiff < 0.1)
-            GlobalStatusStruct.errors +=  _("Low difficulty!; ");
-
-        if(!MinerStatus.ReasonNotStaking.empty())
-            GlobalStatusStruct.errors +=  _("Miner: ") + MinerStatus.ReasonNotStaking;
-
-        unsigned long stk_dropped = MinerStatus.KernelsFound - MinerStatus.AcceptedCnt;
-        if(stk_dropped)
-            GlobalStatusStruct.errors += "Rejected " + ToString(stk_dropped) + " stakes;";
-
-        if(!msMiningErrors6.empty())
-            GlobalStatusStruct.errors +=msMiningErrors6 + "; ";
-        if(!msMiningErrors7.empty())
-            GlobalStatusStruct.errors += msMiningErrors7 + "; ";
-        if(!msMiningErrors8.empty())
-            GlobalStatusStruct.errors += msMiningErrors8 + "; ";
 
+        {
```

Comment:
Did you mean to keep this nested scope? Maybe I'm just tired... I'm not seeing what it's wrapping.

Reply:
Nope. When I moved the locks around a little here I forgot to remove the brace pair, although it doesn't do anything. I will push through a small commit to git rid of it.